open-craft/section-to-course
# fix: Drop autocomplete of courses for performance.

This PR removes the course autocomplete, for performance reasons. It also reduces the number of calls to the modulestore in order to improve performance overall.

**JIRA tickets**: https://tasks.opencraft.com/browse/BB-7424

**Dependencies**: None

**Screenshots**: 

![image](https://github.com/open-craft/section-to-course/assets/1099483/aec67e68-200e-424d-b403-50a4baed7d62)

**Testing instructions**:

1. Follow the directions in this PR, but paste the course key instead of expecting autocomplete
2. Confirm section autocomplete still works
3. Save the course to create the course link and verify the course and courselink have been created.

**Merge checklist:**
Check off if complete *or* not applicable:
